draft-secure-credential-transfer.md

```diff
@@ -42,7 +42,21 @@ author:
     
     
 normative:
-      
+  CCC-Digital-Key-30:
+    author:
+      org: Car Connectivity Consortium
+    title: "Digital Key – The Future of Vehicle Access"
+    date: 2021-11
+    target: https://global-carconnectivity.org/wp-content/uploads/2021/11/CCC_Digital_Key_Whitepaper_Approved.pdf      
+
+
+  ISO-18013-5:
+    author:
+      org: Cards and security devices for personal identification
+    title: "Personal identification — ISO-compliant driving licence — Part 5: Mobile driving licence (mDL) application"
+    date: 2021-09
+    target: https://www.iso.org/standard/69084.html
```

Comment:
I don't think this is the correct standard to link to. This should link to the generic stateless sharing spec, but we have yet to write that spec..